development/editor/creating_icons.rst

```diff
@@ -0,0 +1,83 @@
+.. _doc_editor_icons:
+
+Editor icons
+============
+
+When a new class is created and exposed to scripting, the editor's interface
+will display it with a default icon representing the base class it's inherited
+from. Yet in most cases it is recommended to create icons for new classes
+to improve the user experience considerably.
+
+Creating icons
+~~~~~~~~~~~~~~
+
+In order to create new icons, you first need a vector graphics editor installed.
+For instance, you can use open-source `Inkscape <https://inkscape.org/>`_ editor.
+
+Clone the ``godot-design`` repository containing all the original editor icons:
+
+   .. code:: bash
+
+       git clone https://github.com/godotengine/godot-design
+
+The icons must be created in a vector graphics editor in ``svg`` format. You
+can use ``/engine/icons/inkscape_template.svg`` with default icon properties
```

Comment:
I'd drop the leading `/` which would bring users to their system root if they copy it.